### PR TITLE
Adds Discover help center article

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

Related to https://github.com/elastic/platform-docs-team/issues/371

This PR contains a help center article about Discover. A guided tour within Kibana/Serverless UI about Discover shows some important actions such as sorting on fields, adjusting the table (changing row height), and expanding documents. **This help center article does not repeat the information presented in the guided tour.**

### CODE REVIEW

**Comment by leemthompo** [comment]
@szabosteve might want to mention the ES|QL option here now that it's available on serverless. I'd keep it very minimal, like how to [enable ES|QL in Discover](https://www.elastic.co/guide/en/elasticsearch/reference/8.14/esql-kibana.html#esql-kibana-get-started) and maybe link to ES|QL docs

**Comment by szabosteve** [comment]
@leemthompo Great suggestion, thank you! I'll update the PR in a bit!

File: help-center/discover.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Click ***Save** in the toolbar to save your search configuration and the search
+results. You can use it later to generate a CSV report, create visualizations or
+dashboards. Saving a search saves the query text, filters, and the current view
+of **Discover**, including the columns selected in the document table, the sort
+order, and the data view.
+
+
+## Visualize field values
+
+If the values of a field can be aggregated, you can visualize it from Discover.
```
Comment (KOTungseth) [inline help-center/discover.mdx:121]:
```suggestion
When the values of a field can be aggregated, you can visualize the field in Discover.
```

**Review by KOTungseth — APPROVED** [review]
This looks amazing! I added some comments. Thank you for creating this!

File: help-center/discover.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Adjust the time range to fit your requirements. If your data is not
+time-based, range selection is not available.
+
+To view the count of documents for a given time in the specified range,
+select the corresponding part of the chart.
+
+
+## Explore fields
+
+Discover includes a table that shows all the documents that match your search.
```
[the remark this answers is not in the document]
Comment (szabosteve) [reply help-center/discover.mdx:42]: Good point, I rephrased the first sentence.

**Comment by szabosteve** [comment]
Thank you @KOTungseth for your review and suggestions! I addressed them via [3fc361e](https://github.com/elastic/docs-content/pull/15/commits/3fc361e79af1bae38baa12672c8c6843b0dcae05) and [7c1d870](https://github.com/elastic/docs-content/pull/15/commits/7c1d870e98d00bfbdbcbe7ca3a61b9446254a227).

**Pull request merged**